src/components/Search/EditButton.tsx

```diff
@@ -1,25 +1,27 @@
 import Button from '@mui/material/Button';
 
 import { Strings } from '../../resources/Strings';
+import pencilDark from '../../theme/img/pencil-dark.svg';
+import pencil from '../../theme/img/pencil.svg';
 
 interface EditButtonProps {
   onEditAction: () => void;
+  mode: string;
 }
 
 const baseComp = Strings.components.searchDefault.base;
-const editImgUrl = baseComp.editIcon;
 const editText = baseComp.editText;
 
-const buttonIcon = (
-  <img
-    id='edit-button-icon'
-    className='icon icon-edit'
-    src={editImgUrl}
-    alt='edit-icon'
-  />
-);
+export default function EditButton({ onEditAction, mode }: EditButtonProps) {
+  const buttonIcon = (
+    <img
+      id='edit-button-icon'
+      className='icon icon-edit'
+      src={mode === 'light' ? pencil.toString() : pencilDark.toString()}
+      alt='edit-icon'
+    />
+  );
```

Comment:
I was thinking that it could be easier to use SvgIcon (from Mui), that also has this `color="primary"` attribute that uses the palette directly. But then we'd need to configure how we load SVG [per the documentation](https://mui.com/material-ui/icons/#component-prop) and [this documentation](https://react-svgr.com/docs/webpack/)... Currently we load SVG as `asset/inline`, what makes them a base64 data-url, and this is difficult to use inline because of that.

My general thought is that in SVG we can use `fill=currentColor` that makes it reuse the current color simply.. but that doesn't work if it's inside a `img`, this works only when it's inline (as far as I understand). MUI icons use this trick.

I think this would be good to do one of these eventually but out of scope for this last week.

Reply:
I'll turn this into a separate issue for a unique PR. But I agree.